src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  const shortTerm = 3;
+  const shorTermDiscount = 20;
+  const longterm = 7;
+  const LongTermDiscount = 50;
+  const totalCost = days * dailyRent;
+
+  if (days >= longterm) {
+    return totalCost - LongTermDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return totalCost - shorTermDiscount;
```

Comment:
Correct the referenced constant name to `shortTermDiscount` following the fix of the typo.